samples/msal-node-samples/README.md

```diff
@@ -100,17 +96,17 @@ const config = {
 Once you are in the sample application directory and you've configured the sample application to match your case, you can install all of the dependencies by running:
 
 ```bash
-$ npm install
+    npm install
```

Comment:
@derisen should we ensure they have run `npm run build:all` for the msal-node library before running `npm install` as the msal-node library is defined as a local dependency in the package.json file

Reply:
Good idea -let me add a pointer for setting up msal-node